pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -228,6 +246,20 @@ func (c *constraint) computeSecurityContext(
 		return nil, "", nil, admission.NewForbidden(a, fmt.Errorf("no SecurityContextConstraints found in namespace %s", a.GetNamespace()))
 	}
 
+	if len(requiredSCCName) > 0 {
```

Comment:
since
```go
constraints, err := sccmatching.NewDefaultSCCMatcher(c.sccLister, nil).FindApplicableSCCs(ctx, a.GetNamespace())
```
is effectively `a c.sccLister.List()`, don't you want to simple do a `Get` in this case and save the loop?

Also, the `len(constraints) == 0 { LIST }` here is superfluous. I believe we can safely replace `sccmatching.NewDefaultSCCMatcher(c.sccLister, nil).FindApplicableSCCs(ctx, a.GetNamespace())` with just `c.sccLister.List()`

You can do that in a different commit.

Reply:
I didn't notice https://github.com/openshift/apiserver-library-go/pull/108 before

Reply:
yep

Reply:
108 merged, this is now rebased on top of it